internal/model/modelindexer/indexer.go

```diff
@@ -203,7 +282,9 @@ func (i *Indexer) Stats() Stats {
 		TooManyRequests:       atomic.LoadInt64(&i.tooManyRequests),
 		BytesTotal:            atomic.LoadInt64(&i.bytesTotal),
 		AvailableBulkRequests: atomic.LoadInt64(&i.availableBulkRequests),
-		ActiveBulkRequests:    atomic.LoadInt64(&i.activeBulkRequests),
+		ActiveBulkRequests:    i.scalingInformation().activeIndexers,
```

Comment:
We should revert to how we were tracking the `ActiveBulkRequests` metric before, and maybe expose `activeIndexers` as a new metric.